model-optimizer/unit_tests/extensions/front/tf/CTCGreedyDecoderReplacement_test.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
please use git mv for moving files. Otherwise, we have to review it from scratch.

Reply:
Looks like it is github problem and it shows moved files as deleted and created in new place.